src/app/dataset-view/additional-components/metadata-component/components/form-components/json-kind-field/json-kind-field.component.ts

```diff
@@ -0,0 +1,30 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from "@angular/core";
+import { BaseField } from "../base-field";
+import { FormControl } from "@angular/forms";
+import { ReadKind } from "../../add-polling-source/add-polling-source-form.types";
+import { RadioControlType } from "../../add-polling-source/form-control.source";
+import { SetPollingSourceToolipsTexts } from "src/app/common/tooltips/tooltips.text";
+
+@Component({
+    selector: "app-json-kind-field",
+    templateUrl: "./json-kind-field.component.html",
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class JsonKindFieldComponent extends BaseField implements OnInit {
+    @Input() public data: RadioControlType[];
```

Comment:
I don't like the name `data`. Suggest something clearer. This are control structures that describe radio button choices.

Reply:
Renamed to `controlDescriptors`